src/main/scala/org/clulab/wm/eidos/document/EidosDocument.scala

```diff
@@ -33,18 +32,8 @@ object EidosDocument {
 }
 
 @SerialVersionUID(1L)
-case class TimeStep(startDateOpt: Option[LocalDateTime], endDateOpt: Option[LocalDateTime], duration: Long)
+case class TimeStep(startDateOpt: Option[LocalDateTime], endDateOpt: Option[LocalDateTime])
```

Comment:
Is there a reason to make these `Option`s? I don't think there's any way for them to be `None` is there?

Reply:
For this change I had to update the JLD serialization. Now, intervals is an optional field of timex. @kwalcock is this ok?

Reply:
I don't particularly care, but I don't know who all is using that field, so I should at least ask around first.  The field is documented on https://github.com/clulab/eidos/wiki/JSON-LD, so it has been fair game.

Reply:
@bgyori, do you have an opinion about this?  Should anyone else you know of be asked?  Thanks.

Reply:
@bgyori, we are also about to remove the duration field. Would you want us to keep providing it?

Reply:
It seems I've been successfully confused.  It doesn't look like there is a change in serialization of the timex.  Is it just that the collection of intervals is more likely to be empty than before?  Here's an example of the new version
```json
      "timexes" : [ {
        "@type" : "TimeExpression",
        "@id" : "_:TimeExpression_5",
        "startOffset" : 888,
        "endOffset" : 895,
        "text" : "January"
      }, {
        "@type" : "TimeExpression",
        "@id" : "_:TimeExpression_6",
        "startOffset" : 896,
        "endOffset" : 906,
        "text" : "March 2018",
        "intervals" : [ {
          "@type" : "TimeInterval",
          "@id" : "_:TimeInterval_3",
          "start" : "2018-03-01T00:00",
          "end" : "2018-04-01T00:00"
        } ]
      } ]
```
compared to the old version
```json
      "timexes" : [ {
        "@type" : "TimeExpression",
        "@id" : "_:TimeExpression_8",
        "startOffset" : 888,
        "endOffset" : 895,
        "text" : "January",
        "intervals" : [ {
          "@type" : "TimeInterval",
          "@id" : "_:TimeInterval_8",
          "duration" : 2678400
        } ]
      }, {
        "@type" : "TimeExpression",
        "@id" : "_:TimeExpression_9",
        "startOffset" : 896,
        "endOffset" : 906,
        "text" : "March 2018",
        "intervals" : [ {
          "@type" : "TimeInterval",
          "@id" : "_:TimeInterval_9",
          "start" : "2018-03-01T00:00",
          "end" : "2018-04-01T00:00",
          "duration" : 2678400
        } ]
      } ]
```
The first TimeExpression is missing its TimeInteval as you describe.  The second one isn't.  The durations are indeed gone.  Is this the expected behavior?  Is the TimeInterval gone when there is no specific start and end, since those are now no longer Options?  Things that only had a duration before are no longer included?  That seems fine to me.  Other people may need the details.

Reply:
It does get rid of those pesky things with duration 0! +1

Reply:
"intervals" was a required field. If we removed "duration", the first example in the old version would be:

```      
        "timexes" : [ {
        "@type" : "TimeExpression",
        "@id" : "_:TimeExpression_8",
        "startOffset" : 888,
        "endOffset" : 895,
        "text" : "January",
        "intervals" : [ {
          "@type" : "TimeInterval",
          "@id" : "_:TimeInterval_8",
        } ]
      }
```

I have made "intervals" optional, se we get the output you show in the new version.

Reply:
We do pick up the duration but it's not a big problem if it is removed. Is it always redundant?

Reply:
As far as I know, it is just end - start in seconds and always redundant.  @EgoLaparra, is this true?